src/tests/Microsoft.PowerFx.Repl.Tests.Shared/Modules/basic1_dup.fx.yml

```diff
@@ -0,0 +1,4 @@
+# test symbol conflicts. 
+# Same symbols as base1.fx.yml 
+Formulas: |
+  DoubleIt(x: Number) : Number = x * 200;
```

Comment:
It would be interesting to test diamond dependency like

A -->B--->D
|---> C-----|